multiqc/modules/mirtop/mirtop.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from mirtop"""
+
+from __future__ import print_function
+from collections import OrderedDict
+import logging
+
+from multiqc import config
+from multiqc.plots import bargraph, beeswarm
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+class MultiqcModule(BaseMultiqcModule):
+
+    def __init__(self):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='mirtop',
+        anchor='mirtop', target='Subread mirtop',
```

Comment:
I guess that `Subread mirtop` is a copy and paste error and should just be `miRTop`. `name` can also use the funky caps.